### ISSUE TITLE
Error Page URL

### ISSUE THREAD

**Issue by dreamsh19** [swsnu/swpp2019-team6#41]
지금은 router로 정의되지 않은 url 접근 시 Not Found만 띄우는 페이지로 넘어가는 걸로 아는데 따로 error 페이지를 만들어서 그 error 페이지로 넘어가도록 하는 게 어떨까요? 
백엔드 통신이 실패할 때 넘어갈 수 있는 페이지 주소가 필요할 것 같아서요

**Comment by RXWE** [comment]
네 임시로 Not Found만 표시하는 페이지를 넣어두긴 했는데 따로 에러 띄우는 페이지가 필요할 것 같아요. 그렇지만 우선 순위는 좀 뒤에 둬도 될 것 같네요!

**Comment by deploy-soon** [comment]
네 말씀하신대로 간단하게 500 에러 페이지 만들고 그 쪽으로 보내는 것이 좋을 것 같습니다
